client/src/components/UsedOnTable/tests/UsedOnTable-test.js

```diff
@@ -49,20 +49,6 @@ describe('UsedOnTable', () => {
         expect(mockActions.loadUsedOn).toBeCalledWith('abc', 'get', 'http://www.bob.co.nz');
         expect(mockComponent).toBeCalled();
       });
-
-      it.skip('should try to load new data if identifier changes', () => {
```

Comment:
We shouldn't delete this unit test, it should be adapted to the new lifecycle method

Reply:
This test is currently being skipped altogether. I could try writing an equivalent test, but in practice, I'll mostly just be doubling up the test for `useTabContext`.

Reply:
OK, I didn't notice it was being skipped.  Happy to keep it deleted.